common/network-types/Cargo.toml

```diff
@@ -28,6 +28,7 @@ bitvec = "1.0.1"
 crossbeam-queue = "0.3.11"
 crossbeam-skiplist = { workspace = true }
 dashmap = { workspace = true }
+flume = "0.11.0"
```

Comment:
It'd be better to lift `flume` up as a workspace dep, maybe we might end up using it instead of the `futures::channels`